test/features/operator/install_explainability.feature

```diff
@@ -25,17 +63,18 @@ Feature: Kogito Explainability service
       | username | developer |
       | password | mypass |
     And Install Kogito Trusty with 1 replicas with configuration:
-      | infinispan | username | developer                 |
-      | infinispan | password | mypass                    |
-      | infinispan | uri      | external-infinispan:11222 |
-      | kafka | externalURI | external-kafka-kafka-bootstrap:9092 |
-    And Install Kogito Explainability with 1 replicas with configuration:
+      | infinispan | username    | developer                           |
+      | infinispan | password    | mypass                              |
+      | infinispan | uri         | external-infinispan:11222           |
+      | kafka      | externalURI | external-kafka-kafka-bootstrap:9092 |
+
+    When Install Kogito Explainability with 1 replicas with configuration:
       | kafka | externalURI | external-kafka-kafka-bootstrap:9092 |
+    And Clone Kogito examples into local directory
     And Local example service "dmn-tracing-quarkus" is built by Maven using profile "default" and deployed to runtime registry
     And Deploy quarkus example service "dmn-tracing-quarkus" from runtime registry with configuration:
-      | config      | enableEvents        | enabled                                    |
-      | runtime-env | KOGITO_SERVICE_URL  | http://dmn-tracing-quarkus:8080            |
```

Comment:
I know that BDD tests on kubernetes have some bugs, for example multiple overlapping resources are created -> the tests fail. Btw the config ` | runtime-env | KOGITO_SERVICE_URL  | http://dmn-tracing-quarkus:8080            |` is needed in case somebody runs the bdd tests on kubernetes and not on openshift. 
Just fyi

Reply:
Then this sounds like a bigger bug that we should fix in the test framework. I will run the test on Kubernetes and provide a JIRA if I can reproduce it. 
Thanks!

Reply:
Yeah, unfortunately this is what I faced when multiple services are exposed (for example data-index + kogito runtime or trusty + runtime etc..). 
Another thing is that there is an hardcoded port `80` instead of `8080`, which is the default port used by the services

Reply:
The port bug is here https://github.com/kiegroup/kogito-cloud-operator/blob/67fc59bc10abb885ae97f8a48ad146c538c48215/test/framework/kubernetes.go#L401 
And in the same file the issue is the function `ExposeServiceOnKubernetes` which creates overlapping ingress resources

Reply:
Ok, thanks! Maybe this logic is out to dated and as we don't run the tests against Kubernetes yet, we didn't spot it yet.